### DISCUSSION TITLE
Enhance the c-lightning node operation

### DISCUSSION THREAD

**Discussion by raduButucelea23** [btcpayserver/btcpayserver#4381] [Ideas / Feature Requests]
Although "lightning node backups" are still a WIP topic with nothing standardized yet, I would like to suggest a list of upgrades to enhance the reliability of a c-lightning node operation in the context of BTCPay Server with docker.

1. create a c-lightning seed backup service to enable recovery of on-chain funds (similar to LND seed backup). Ref. to [documentation](https://lightning.readthedocs.io/BACKUP.html#hsm-secret)
    - #4385
3. update the c-lightning implementation release to v0.12.x. 
    - ElementsProject/lightning#5657
    - btcpayserver/btcpayserver-docker#664
    - btcpayserver/btcpayserver-docker#647

**Comment by dennisreimann** [comment]
Added links to the individual issues where these are tracked.

**Comment by bigsk1** [comment]
agreed been running c-lightning for over 3 years on btcpay with zero chance of a backup if all goes wrong.
